### ISSUE TITLE
Minikube errors coming while starting access denied 

### ISSUE THREAD

**Issue by pappusanodiya** [kubernetes/minikube#19814]
### What Happened?

C:\Users\pspap>minikube start
* minikube v1.34.0 on Microsoft Windows 11 Home Single Language 10.0.22631.4317 Build 22631.4317
* Using the virtualbox driver based on existing profile
* Starting "minikube" primary control-plane node in "minikube" cluster
* Restarting existing virtualbox VM for "minikube" ...
! StartHost failed, but will try again: driver start: Too many retries waiting for SSH to be available.  Last error: Maximum number of retries (60) exceeded
* Updating the running virtualbox "minikube" VM ...
* Failed to start virtualbox VM. Running "minikube delete" may fix it: provision: Error getting config for native Go SSH: open C:\Users\pspap\.minikube\machines\minikube\id_rsa: Access is denied.

X Exiting due to GUEST_PROVISION: error provisioning guest: Failed to start host: provision: Error getting config for native Go SSH: open C:\Users\pspap\.minikube\machines\minikube\id_rsa: Access is denied.

### Attach the log file

![image](https://github.com/user-attachments/assets/92b48cf9-dd16-4529-ba12-7e1b0e8e5c4f)

### Operating System

Windows 11 

### Driver

Virtualbox

**Comment by pappusanodiya** [comment]
![image](https://github.com/user-attachments/assets/0bb6b00e-d1d6-49a7-ac3f-3e955f6d738e)

**Comment by kundan2707** [comment]
@pappusanodiya 
You don't seem to have permission to access to your id_rsa file.
Are you using Command Prompt or Powershell? And did you Run as administrator when you opened it?
Run with administrator so that you will have full permission.

**Comment by pappusanodiya** [comment]
Yes, I have tried to run with administration, kubectl cmd is working but this error is coming while starting or stoping minikube

**Comment by pappusanodiya** [comment]
Here running with administrators, working fine and with the normal command prompt showing access denies so for working in minikube every time do I need to run with administrator command prompt or we can give the access power to run with normal command prompts 

![image](https://github.com/user-attachments/assets/68c19d8b-1024-4e45-8ac3-e96047119611)
